### DISCUSSION TITLE
Cant Access Ollama at port 8888

### DISCUSSION THREAD

**Discussion by elhakimz** [open-webui/open-webui#2128] [General]
# Bug Report

## Description
Cant access ollama host at localhost:8888

**Bug Summary:**
[Provide a brief but clear summary of the bug]
![Screenshot 2024-05-09 094646](https://github.com/open-webui/open-webui/assets/616176/289e172f-7c88-4543-8d90-d1463ecdb4a6)

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]
docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:8888 --name open-webui --restart always ghcr.io/open-webui/open-webui:main

**Expected Behavior:**
[Describe what you expected to happen.]
Ollama should be accessible

**Actual Behavior:**
[Describe what actually happened.]
![Screenshot 2024-05-09 094646](https://github.com/open-webui/open-webui/assets/616176/0ef75859-fe4d-4c4f-b6d0-3c0151c1c017)

## Environment

- **Open WebUI Version:** [e.g., 0.1.120]
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by justinh-rahb** [comment]
Unless you've also set `OLLAMA_HOST=127.0.0.1:8888` for Ollama, you'd need to use `-e OLLAMA_BASE_URL=http://127.0.0.1:11434` by default.
